scripts/exregional_run_fcst.sh

```diff
@@ -106,7 +106,9 @@ case $MACHINE in
   "WCOSS_DELL_P3")
     ulimit -s unlimited
     ulimit -a
-    APRUN="mpirun -l -np ${PE_MEMBER01}"
+    APRUN="mpirun"
+    OMP_NUM_THREADS=4
```

Comment:
This will also change the RRFS DA cycle performance. We need test it.

Reply:
Shun is running RRFS using OMP_NUM_THREAD=1. We need to check with Shun on this because his layout also based on this. This could speed up the forecast. We need to tell Shun first.

Reply:
@ShunLiu-NOAA

Reply:
I will revert this change to get this PR in so as to minimize the differences between RRFS and RTMA.